backend/src/packages/trucks/truck.service.ts

```diff
@@ -66,6 +68,32 @@ class TruckService implements IService {
 
     return result.map((element) => TruckEntity.initialize(element).toObject());
   }
+
+  public async addTrucksToUser(
+    userId: number,
+    trucksId: number[],
+  ): Promise<number[]> {
+    return await Promise.all(
```

Comment:
Promise.all can exhaust database connections pool, better to avoid it when dealing with database queries